### PR TITLE
upvote interface 

### PR DESCRIPTION
**Pull request by DanielNugent**
## Purpose

Adding upvote downvote UI and potentially fixed camera buttons not showing on smaller screens (I cant verify this myself as it doesn't work on web anymore) and changed save image to an icon for more screen space
Adding loading screen
partial fix to camera buttons not showing (needs more work :x)

## Checklist

- [x] Add upvote/downvote UI
- [x] Add loading state when waiting for image
- [ ] Fix camera buttons not showing on small screens (@dabreadman can you check this and see if it works, aspect ratio might be wrong, however)

### CODE REVIEW

File: screens/ChooseStyleScreen.tsx [hunk 1]
```diff
@@ -18,16 +19,29 @@ type Props = NativeStackScreenProps<RootStackParamList, 'ChooseStyleScreen'>
...
 
-  const handleContinue = () => navigation.navigate('FinishedArtScreen', { image: image })
+  const handleContinue = () => {
+    setLoading(true)
+    navigation.navigate('FinishedArtScreen', { image: image })
+  }
 
   return (
     <View style={styles.container}>
+      <SnackBar
```
Comment (DanielNugent) [inline screens/ChooseStyleScreen.tsx:45]: I added a snackbar for displaying errors when posting to the backend, maybe lex can use it

Comment (dabreadman) [reply screens/ChooseStyleScreen.tsx:45]: image

Comment (DanielNugent) [reply screens/ChooseStyleScreen.tsx:45]: thanks, ill try it again i guess

**Review by ClaireCassidy — DISMISSED** [review]
Looks great on my device!

**Review by lexesj — DISMISSED** [review]
Looking good! Thanks for adding the loading components, will make use of them once this PR is merged in.

**Review by johnmarksinclair — DISMISSED** [review]

**Review by lexesj — APPROVED** [review]

**Pull request merged**